media/src/main/java/io/wcm/handler/media/util/ImageUtils.java

```diff
@@ -0,0 +1,96 @@
+/* Copyright (c) pro!vision GmbH. All rights reserved. */
+package io.wcm.handler.media.util;
```

Comment:
this creates a new package that is exported by OSGi - but i suppose the util class is not intended for public usage (we can always make it public later).
recommendation: move the class to {{io.wcm.handler.media.ui}} and make it non-public there.

Reply:
good point. I moved it